.env.example

```
@@ -1,2 +1,4 @@
 TURSO_DATABASE_URL=""
-TURSO_AUTH_TOKEN=""
\ No newline at end of file
+TURSO_AUTH_TOKEN=""
+SUPABASE_SERVICE_KEY=""
+SUPABASE_URL=""
```

missing new line here
```suggestion
SUPABASE_URL=""

```